lib/3scale_toolbox/commands/import_command/import_csv.rb

```diff
@@ -163,11 +152,10 @@ def self.import_csv(destination, file_path, insecure)
           puts "#{stats[:mapping_rules]} mapping rules have been created"
         end
 
-        def self.run(opts, _)
-          destination = fetch_required_option(opts, :destination)
-          file_path = fetch_required_option(opts, :file)
-          insecure = opts[:insecure] || false
-          import_csv(destination, file_path, insecure)
+        def run
+          destination = fetch_required_option(:destination)
+          file_path = fetch_required_option(:file)
+          import_csv(destination, file_path)
```

Comment:
The same comment in some other command: IMO declaring variables and running the actual action would be nice to be separated by a newline.